src/Bybop_Device.py

```diff
@@ -430,6 +430,23 @@ def emergency(self):
         An emergency request shuts down the motors.
         """
         self.send_data('ardrone3.Piloting.Emergency')
+    def roll(self,roll_value):
+        """
+        Have the drone move left or right depending on the roll_value. 
+        Refer to here for more information: https://github.com/Parrot-Developers/arsdk-xml/blob/e73425074471c58561d04c85da4a6400b638779d/xml/ardrone3.xml#L66
+        NOTE: No acknowledgement is returned.
+        """
+        if((roll_value >100) or (roll_value < -100)):
+            print("Drone Roll value must be between -100 and 100")
+            return
+        self.send_data('ardrone3.Piloting.PCMD',(0,roll_value,0,0,0,0))
```

Comment:
As stated in [the documentation](http://developer.parrot.com/docs/reference/bebop_2/#move-the-drone), you'll have to se the flag (1st argument) to 1 when setting a non-zero roll (or pitch) value.

Another problem is that the drone expects the PCMD to be periodic, and will revert to a "zero" (hovering) command if no PCMD is recieved in the last 500ms (in current firmwares). That's why we have an internal send loop in `libARController` for this command (and other periodic commands)

All in all, developing a proper `drone.set_pcmd(...)` command would require more work.